src/directives/formBuilderOptionAddAnotherPosition.js

```diff
@@ -0,0 +1,36 @@
+/**
+* A directive for a datagrid to edit a component's button position.
+*/
+module.exports = function() {
```

Comment:
Why did you have to create a new directive for this? We have other options that use select dropdowns that do not require you to create a new directive. Please see how those were implemented.

Reply:
I was under the impression that each select dropdown menu required its own directive. When creating this one, I reviewed:

https://github.com/formio/ngFormBuilder/blob/master/src/directives/formBuilderOptionLabelPosition.js
and
https://github.com/formio/ngFormBuilder/blob/master/src/directives/formBuilderOptionOptionsLabelPosition.js

My understanding was that each select dropdown requires a unique controller to define the value and titles of each menu item and to set a default.